cloudify_cli/utils.py

```diff
@@ -168,6 +172,11 @@ def is_initialized():
 
 
 def get_init_path():
+    """
```

Comment:
according to PEP8, docstrings should be constructed as follows:

```python
"""A single liner describing what this does/returns

more elaborate explanation
"""
```

the format is as specified above.